assets/js/googlesitekit/widgets/datastore/widgets.js

```diff
@@ -71,17 +71,19 @@ export const actions = {
 	 * Registers a widget with a given slug and settings.
 	 *
 	 * @since 1.9.0
-	 * @since 1.12.0 Added wrapWidget setting.
+	 * @since 1.12.0  Added wrapWidget setting.
+	 * @since n.e.x.t Added hideOnBreakpoints setting.
 	 *
-	 * @param {string}                slug                   Widget's slug.
-	 * @param {Object}                settings               Widget's settings.
-	 * @param {WPComponent}           settings.Component     React component used to display the contents of this widget.
-	 * @param {number}                [settings.priority]    Optional. Widget's priority for ordering (lower number is higher priority, like WordPress hooks). Default is: 10.
-	 * @param {string|Array.<string>} [settings.width]       Optional. Widget's maximum width to occupy. Default is: "quarter". One of: "quarter", "half", "full".
-	 * @param {boolean}               [settings.wrapWidget]  Optional. Whether to wrap the component with the <Widget> wrapper. Default is: true.
-	 * @param {string|Array.<string>} [settings.modules]     Optional. Widget's associated modules.
-	 * @param {Function}              [settings.isActive]    Optional. Callback function to determine if the widget is active.
-	 * @param {Function}              [settings.isPreloaded] Optional. Callback function to determine if the widget should be preloaded if not active (requires isActive).
+	 * @param {string}                slug                         Widget's slug.
+	 * @param {Object}                settings                     Widget's settings.
+	 * @param {WPComponent}           settings.Component           React component used to display the contents of this widget.
+	 * @param {number}                [settings.priority]          Optional. Widget's priority for ordering (lower number is higher priority, like WordPress hooks). Default is: 10.
+	 * @param {string|Array.<string>} [settings.width]             Optional. Widget's maximum width to occupy. Default is: "quarter". One of: "quarter", "half", "full".
+	 * @param {boolean}               [settings.wrapWidget]        Optional. Whether to wrap the component with the <Widget> wrapper. Default is: true.
+	 * @param {string|Array.<string>} [settings.modules]           Optional. Widget's associated modules.
+	 * @param {Function}              [settings.isActive]          Optional. Callback function to determine if the widget is active.
+	 * @param {Function}              [settings.isPreloaded]       Optional. Callback function to determine if the widget should be preloaded if not active (requires isActive).
+	 * @param {string|Array.<string>} [settings.hideOnBreakpoints] Optional. Hide widget on selected breakpoints. One of: "BREAKPOINT_SMALL", "BREAKPOINT_TABLET", "BREAKPOINT_DESKTOP", "BREAKPOINT_XLARGE".
```

Comment:
The documentation for this param is a bit off, but let's also change it to only be an array. I think it gets a bit confusing when the argument is plural (eg. implied to be an array) but it also accepts a single value.

```suggestion
	 * @param {Array.<string>} [settings.hideOnBreakpoints] Optional. Hide widget on selected breakpoints. Array with any of: `BREAKPOINT_SMALL`, `BREAKPOINT_TABLET`, `BREAKPOINT_DESKTOP`, `BREAKPOINT_XLARGE`.
```